dev/buildtool/container_commands.py

```diff
@@ -142,13 +145,18 @@ def _do_command(self):
             logging.info("Tagging container: %s(-ubuntu)", existing_image)
 
             for tag in tag_permutations:
-
                 alpine_image = f"{SPINNAKER_DOCKER_REGISTRY}/{service}:{tag}"
                 self.regctl_image_copy(existing_image, alpine_image)
 
                 ubuntu_image = f"{SPINNAKER_DOCKER_REGISTRY}/{service}:{tag}-ubuntu"
                 self.regctl_image_copy(f"{existing_image}-ubuntu", ubuntu_image)
 
+                alpine_image_java11 = f"{SPINNAKER_DOCKER_REGISTRY}/{service}:{tag}-java11"
+                self.regctl_image_copy(existing_image_java11, alpine_image_java11)
```

Comment:
How does this work if existing_image_java11 doesn't exist?

Reply:
It'll fail. 
I was thinking we would have done a round of tagging but actually that'll only work for new releases. 

For releasing older versions we will need to pass in a parameter to enable tagging Java 11 or not 😞

Reply:
Does it fail immediately at that point or just print the error and continue?

Reply:
It will fail immediately at the regctl command I think as regctl will return non zero exit code. 
You can try using regctl manually to tag an image that doesn't exist. 

It seems the buildtool scaffolding around executing commands is robust. Any non-zero exit code results in an exception, log and program halt.

We could continue on instead which would be okay for old minor releases but not ideal for new.

Reply:
I don't run a Mac so am not sure about running tests locally. 
1. It may be related to the Bash version? I've been running them on linux with Bash version 4. Mac has v3 by default IIRC.
2. It could be other file presence checks like [git_support_test.py](https://github.com/spinnaker/buildtool/blob/a780b26f4c78d1651f7c11c378ec9e50f77248e2/unittest/buildtool/git_support_test.py#L527)?

For now using CI and the GHA draft release action should be ok.

Reply:
Seems to be an issue with `git`. 

`buildtool.errors.ExecutionError: git failed.`